sunpy/net/jsoc/attrs.py

```diff
@@ -71,6 +75,45 @@ def __init__(self, value):
         self.value = value
 
 
+class Cutout(DataAttr):
+    """
+    Select a cutout region to be processed by JSOC
+
+    Parameters
+    ----------
+    bottom_left
+    width
+    height
+    tracking
+    register
+    nan_off_limb
+    """
+
+    @u.quantity_input
+    def __init__(self, bottom_left, top_right=None, width: u.arcsec = None,
+                 height: u.arcsec = None, tracking=False, register=False,
+                 nan_off_limb=False):
+        super().__init__()
+        bl, tr = get_rectangle_coordinates(bottom_left, top_right=top_right, width=width,
+                                           height=height)
+        self.value = {
+            't_ref': bl.obstime.isot,
+            # JSOC input is disable tracking so take the negative
+            't': int(not tracking),
+            'r': int(register),
+            'c': int(nan_off_limb),
+            'locunits': 'arcsec',
+            'boxunits': 'arcsec',
+            'x': ((bl.Tx + tr.Tx) / 2).to('arcsec').value,
+            'y': ((bl.Ty + tr.Ty) / 2).to('arcsec').value,
+            'width': (tr.Tx - bl.Tx).to('arcsec').value,
+            'height': (tr.Ty - bl.Ty).to('arcsec').value,
```

Comment:
I think you need to take all four corners of the box and calculate the maximum width and height in world space for that rectangle. For instance, if the world rectangle here was rotated Westward on the disk then your maximum width would be between the Tx coord of the top left and the bottom right, not the top right and bottom left as calculated here.